tests/test_views.py

```diff
@@ -36,3 +36,117 @@ def test_detailed_records(self):
             except Exception:
                 failing_recids.append(recid)
         self.failIf(failing_recids)
+
+    def test_literature_landing_page(self):
+        """Test visiting each landing page returns 200"""
+        failing_landing_page = []
+        try:
+            if self.client.get("/literature").status_code != 200:
+                failing_landing_page.append('/literature')
+        except Exception:
+            failing_landing_page.append('/literature')
+        self.failIf(failing_landing_page)
```

Comment:
Just a curiosity here, if an exception happens, will it be printed? (since it is caught before `failIf`)